tests/BuildSystem/Build/changed-commands.llbuild

```diff
@@ -18,8 +18,8 @@
 
 ##### VERSION-BEGIN-1 #####
 
-# CHECK-VERSION-1: echo output-1 > output-1
-# CHECK-VERSION-1: cat output-1 > output
+# CHECK-VERSION-1: echo\ output-1\ \>\ output-1
+# CHECK-VERSION-1: cat\ output-1\ \>\ output
```

Comment:
ah.. it ends up being unfortunate for the readability of our test cases... another reason to use single-quoted escaped style instead of inline `\\` escapes